app/server.go

```diff
@@ -26,6 +26,7 @@ import (
 	"github.com/getsentry/sentry-go"
 	sentryhttp "github.com/getsentry/sentry-go/http"
 	"github.com/gorilla/mux"
+	"github.com/mailru/easygo/netpoll"
```

Comment:
This means that we are dropping support for Windows NT right, don't know if we ever support it, but just highlighting this

Reply:
We do actually. I never realized. I will talk with PM.